<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createGroupDlg</class>
 <widget class="QDialog" name="createGroupDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>创建群聊</string>
  </property>
  <property name="windowIcon">
   <iconset resource="TinyChat.qrc">
    <normaloff>:/images/images/bitbug_favicon.ico</normaloff>:/images/images/bitbug_favicon.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="6,1">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
      <item>
       <widget class="QWidget" name="widget_friend_info" native="true">
        <property name="styleSheet">
         <string notr="true">border: 1px solid rgb(224,238,238)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_selected_friend" native="true">
        <property name="styleSheet">
         <string notr="true">border: 1px solid rgb(224,238,238)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_create_group" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,1">
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_group_name">
           <property name="placeholderText">
            <string>请输入群名称</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="button_create_group">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>确认</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="TinyChat.qrc"/>
 </resources>
 <connections/>
</ui>
